### PR TITLE
Fixes #379 Update package.json file

### LINKED ISSUE #379: Update package.json
Since we are updating our code, and making it cleaner, let us also update package.json, add the line to deploy in it? 
This was discussed in [#198](https://github.com/fossasia/susper.com/pull/198) and [#210](https://github.com/fossasia/susper.com/pull/210)

### PR DESCRIPTION
**Pull request by harshit98**
Closes #379 

Preview link (if needed) : <a href="https://harshit98.github.io/susper.com/">here</a>.

Things done in this PR : 
- Updated package.json file as per issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "test": "ng test",
     "pree2e": "webdriver-manager update",
     "e2e": "protractor",
-    "postinstall": "ng build --prod --aot"
+    "postinstall": "ng build --prod --aot",
+    "deploy": "ng build --prod --aot --base-href=/susper.com/ && cp ./dist/index.html ./dist/404.html && ./node_modules/.bin/angular-cli-ghpages --no-silent"
```
Comment (AnshulMalik) [inline package.json]: Do we have same error page as index ?

Comment (harshit98) [inline package.json]: @AnshulMalik I did not get it. Kindly elaborate. :) There was only a deploy script missing which needs to be added. That's it.

**Comment by harshit98** [comment]
@Marauderer97 done! :)

**Comment by harshit98** [comment]
@nikhilrayaprolu @Marauderer97 I have updated all the packages. Now Angular v2.3.0 has been bumped to Angular v4.1.3. If some issues will occur then there will be some changes in - 
- `typescript v2.1+` which also has been upgraded.
- change access specifier from `private` to `public` 

There will be some issues which we will be going to face. So, we'll open up those issues separately, if they happen.

**Comment by nikhilrayaprolu** [comment]
have you updated your demo link? so that we could test for breaking changes there?
secondly have you gone through breaking changes log in https://github.com/angular/angular/blob/master/CHANGELOG.md

**Comment by harshit98** [comment]
@nikhilrayaprolu Updated Preview link - <a href="https://harshit98.github.io/susper.com/">here</a>. Yes, I have gone through the log but didn't found any errors. No errors in deploying, testing and hosting a local server.

**Pull request merged**